permissions/plugin-xlrelease-plugin.yml

```diff
@@ -4,4 +4,7 @@ paths:
 - "com/xebialabs/ci/xlrelease-plugin"
 developers:
 - "jdewinne"
-- "mbigorne"
\ No newline at end of file
+- "mbigorne"
+- "msaraswat"
```

Comment:
does not exist https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=msaraswat